src/pages/City/AddAdministratorModal/AddCitiesSecretaryForm.tsx

```diff
@@ -83,7 +83,7 @@ const AddCitiesNewSecretaryForm = (props: any) => {
   }, [props]);
 
   return (
-    <Form name="basic" onFinish={handleSubmit} form={form} className="formAddSecretaryModal">
+    <Form name="basic" onFinish={(values) => {handleSubmit(values); setLoading(true)}} form={form} className="formAddSecretaryModal">
```

Comment:
Where do you use `setLoading(false)` after this action is done?

Reply:
in button

Reply:
it sets button disabled(in loading)